web/pdf_viewer.js

```diff
@@ -1992,43 +1992,63 @@ class PDFViewer {
 
   /**
    * Increase the current zoom level one, or more, times.
-   * @param {number} [steps] - Defaults to zooming once.
    * @param {Object|null} [options]
    */
-  increaseScale(steps = 1, options = null) {
+  increaseScale(options = null) {
     if (!this.pdfDocument) {
       return;
     }
-    let newScale = this._currentScale;
-    do {
-      newScale = (newScale * DEFAULT_SCALE_DELTA).toFixed(2);
-      newScale = Math.ceil(newScale * 10) / 10;
-      newScale = Math.min(MAX_SCALE, newScale);
-    } while (--steps > 0 && newScale < MAX_SCALE);
 
     options ||= Object.create(null);
-    options.noScroll = false;
+
+    let newScale;
```

Comment:
Why not just do the following here, to simplify the code below?
```suggestion
    let newScale = this._currentScale;
```